lms/templates/video.html

```diff
@@ -170,6 +170,19 @@ <h4 class="hd hd-5">${_('How is the transcript quality ?')}</h4>
             </div>
         </div>
         % endif
+        <div class="google-disclaimer">
+            <span className="text-dark-300 x-small">Powered by
+                <a href="https://translate.google.com/" target="_blank">
+                    <img
+                        width="100"
+                        className="inline-image pl-1"
```

Comment:
I think this would work for the `Image` component from `edx-paragon`, for the `<img>` tag I think it doesn't apply. But now that I check, the whole `classname` is unnecessary.